frontend/src/components/RecordsList.tsx

```diff
@@ -257,22 +261,20 @@ export default function RecordsList({
         </Modal>
       )}
 
-      {samplesQueryParam && (
+      {Object.keys(params).length !== 0 && (
```

Comment:
`params` returns `{}` when the relative URL is (for example) `/requests`.

It returns `{ igoRequestId: 16067_C }` when the relative URL is `/requests/16067_C`. 

By checking its length, we can know whether we should display the Request/Patient/Cohort Samples view.